op-node/p2p/peer_scorer.go

```diff
@@ -43,18 +44,50 @@ func NewScorer(peerGater PeerGater, peerStore Peerstore, metricer GossipMetricer
 	}
 }
 
+// ScoreBands holds the number of peers in each peer score band.
+// This is intended to be used with a [Metricer].
+type ScoreBands struct {
+	Graylist int
+	NoGossip int
+	NoPX     int
+	Friend   int
+}
+
+// Record records a peer score in the appropriate band.
+func (s *ScoreBands) Record(score float64) {
+	thresholds := NewPeerScoreThresholds()
+	if score <= thresholds.GraylistThreshold {
+		s.Graylist = s.Graylist + 1
+	} else if score <= thresholds.GossipThreshold {
+		s.NoGossip = s.NoGossip + 1
+	} else if score <= thresholds.AcceptPXThreshold {
+		s.NoPX = s.NoPX + 1
+	} else {
+		s.Friend = s.Friend + 1
+	}
+}
```

Comment:
I'd prefer to have a non-hard-coded scores band config. E.g. we could have a config parameter `OP_NODE_METRICS_P2P_SCORE_BANDS` of the format `<threshold>:<label>;...` like `-40:graylist;0:no_gossip;...` and then read this out during setup of the metrics. Should require that thresholds are ordered.

Reply:
OK I'm fine with not doing it in this PR but please create a ticket for it in Linear. Right now, with these static bands, if we look at the metrics and want to adjust thresholds, we have to go through the full cycle of opening a PR here, create a release and get it deployed. I would like to avoid these cycles by just making them configurable in the first place. Shouldn't be much work. Just parsing a config string and working on a `map[int]int` and `[]int` of thresholds instead of the static struct.

Reply:
Will just impl